python/instrumentation/openinference-instrumentation-bedrock/tests/test_instrumentor.py

```diff
@@ -23,6 +24,14 @@
x99B\x82\x93\xc9\xc6U\xd5n\xe9\xec\xf8\x9b,\xa1\x88\xe3\xa7\x9cE\xee\xd6\x17r\x89e.\xef\xb7j\x9c\x93v;)\xc0\xb1:\x87q\xe1Y\xea\xfa\r\xc0\x8bf\xe3d\x8fL\xcf\x16\xb70\xac\xe7\xe9\xa7\xad\xec\\\x07\xa4\xe9\x02\x9c\xe5l\x7fD\x0c\xd7\xc5f\x86\x87\xdcw\xb6\xe4\x82\xef\xfc!\x94 \xbb\xc90\x00j~[\x84E\x88D\xd7\x18\xec\x03\xe1\xd4\x91\xe9S\x05\x10\xd3+{Y\x89\xaf\xdf%V\xb9<\xf6%s}d\xf2S\xab\xf6\xc2\x03MG\xfb\xb5%\xcdo\x07\xdf\xd1_(f-\xff)\xca\x8a\xeb\xaa\xe6\xafMb\xaa!b4\x99Y\x1e\xeb\\\xfeDoXB\xe7`\xaeq1\xa4\xe3\x8a\x00\xe6\xc8l\x96\xee/N\xd3\x89|\xec\x01m/\xaa\x03q\xfd\xa0W_\x98\x0b1\xc2\x04\xc1\x1cd\xc8\xd3\x8dT\x05\x16\xee\xe5p\x02\xa9\xa1\x9a\xef-\x01\"\xeb\xe6\xca$,\x1d\x15\xb2\x03D\xe9\x92\xb3\xc2O\xec`\xe9\xad\xc6\xad\x03\xfd\x8f\xe5\xab(]\xca\t\xae\xa7\x19\x8d\xa2\x11\xcf\x8e\xf5Y\x04\x13O?\xdbc\r\xd7\x8c\xa6\x04\xf7\xe7?\x9a\x9e>w?\x96\x83\x0f\xa1`0O\xfa\xabX\x84W\xa2\x0bt\xf6F\xd9\xb6\xdb\xd1o\xb46\x88\xc0/\xec\xb0\xe3\xa9\xd8J\xa6\x85\xa1\xa9\xbes\xeb\x10)\xfe\x8f\xb5k5\x0f$\xf19\x08\x8dU\xe0\":\xac\xdbL\xd0&\x05\x95}+\x01\xb3\xd3\x01M\x03=\xb2>\xf6\x10\xd9\x1fw\xea\xea\xe5\xbf0\x12/G\x98M\xd3\x1d\x87\xfb\xa5\xea\xbf\xef\xba\xa4\xe1\x18\x97\x8bp\x073\xf9\x9fPJL\x83u\x95\x81\xf2\x0b~5L\xc2*{\xf8_\xed}\xff\x8bA\xae\x06j\xb6\xealbq>,\r\xdcymD\x9eC\xc9\xce&l\xde\xbd%\xf4\xbe\x830N\xb5\xadX\xe1G\xa0\x03\x89\xf4\xc7\xe5\x9b\x0e\x0f\x87H~\rf{\x9cU\x08f^}\xec\xbd\xab\x00\x1b\xe3\xdb9_\x19\xd5\xd5\x0b\xc3\xceT\xbf\x06\x8f\xdd\xb2\\\xf0\xc0\xa7\x15A\xdd\xbe'\r\xc7\xfd\x9eb\x06\x10Uz\x133n\x06|\x8f\n\xb6\x13PvI\xb0\xd2\xae\xed\x0fUpjL\xc8Ro\xba\x98y\xb8\xaf\xbdbT\xceDK\xbe\\\xbf\x08\xa8\x8c1s\xbf\xe3I\xdbJP\xf54c\x13\xcaH\x9c>)C:\xd0\x14\xecH\x02g\x17\x08ON\xef\x81/\xbd\xe7F\x19\x98\x95\x0f\x03\x98\xdc\x15\x9c\xcd}\xd9Z\x1c\x10\x11\x9c\xf7=\xfe`^S~\xc1d\xfb9\xe5\x10x<\xda\xaf\x01A\xee\xed\xb0\x80\x9a\x05\xa3\xf6p\xd2\x9b\xba\x14\n\x99\xb7\xdf\xe1\xac\x8b,#\x97\xea/\xa3G\x1a\x00xZ\xca\xef& \x90O$\xbd\xe9\x8e\x00\x11\x8e\x8c\x0c\xf1\xd2b\xa7\x83[T\x90R\xe5n\xcd)Y\x01\x9b\x12\xf5V\xcc6t\xfa\xb1\xf4\x18\x12\xa7h\xba\xaa\xb3\xe4`\xf3\x00\xbez\xc0\xb7\xb4\x87\x9c\x10>\xbcs8\nbN\x8b\xe7\x12\xe1\xde\xf0\x95\x98\xc0\xa8\x97\xd2Gv\xec\xca\x01ZTlB\xa2\xab.\xb8\xd2\x0e\xdf5\x1a\x84\xf5M`\xc8\xd5\xca0\x8f7\xbbB\xa4\xcb\xe3u\x00\xa3\xea~)\xa8\x02fj\xee\xea\x08\xf6\\\xe0l\xfd$`y\x85\xae\xeb\x0c\x0c%\x91i\xc7\xc7\x08\x0eZ\x02\xce\xcc\xb6\x17\x95\n\xbf;\x12\xe7\xa97\x87\xf8\x91xz\x10:\x90\xcfM\xca\xe5O\x01\xa0\x00AI\x85\x8bnV\x9c\xc9\xc1W\x17\xb1SL\x91\x8c\xe6\x1d\x97\xf1\xfaczX\xe9\xdf\x05\xec\xca\x8b\xad\xd7\x89\xd7\x1aI\x0f@l\xd3\xc6n\xa1\x1fZ\x10\x9aC\xe5\xba\xc1h\x81\x15\xf9@\x07\x1e\x1f^%(\xac\x12s\xc0$\xa7\xa1S\xa7\xa8=\x16l\x16\x83\xd0\x12\"\x14k\xe6VH\x18\x8c\x11\xc1\xcd\xcc9 \xdd\xd8\xf8\x15\x1e\xef\xaaW\x91\xc5\xfc\x97{4\xa0dM\x01\xc2CrZ\x0e\x8a\xf7\xc4\xa0\xb4\xe9\x91,y\xc4eYV\x99\x8d\xdbY\x80\x02\x00\x1c= /Z/\x94\x98gOr9\xb9\xc4\xd4\xf8Y]\xb9!\xb5\xf2\x17a\xac7F\r\xc1\x16a\x96\xa9Ic\xdd\xfd\xf2\x06\xde\x94N\r\x85\xb3;R|WK\xa3\xc8\xc0\x00\x00\x00",  # noqa: E501
```

Comment:
do we prefer to put a URL and dowload and get the bytes from it? hard coding the byte array it's kind of ugly

Reply:
you can use this 1px gif
bytes
```python
b'GIF89a\x01\x00\x01\x00\x80\x00\x00\xff\xff\xff\x00\x00\x00!\xf9\x04\x01\x00\x00\x00\x00,\x00\x00\x00\x00\x01\x00\x01\x00\x00\x02\x02D\x01\x00;'
```
base64
```python
"R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=="
```

Reply:
Done